lib/virt_autotest/utils.pm

```diff
@@ -85,19 +104,45 @@ sub restart_modular_libvirt_daemons {
     } else {
         # Restart the sockets first
         foreach my $daemon (@daemons) {
-            systemctl("restart $daemon\{,-ro,-admin\}.socket");
+            if ($daemon == 'lock') {
```

Comment:
It seems that there are ```virtproxyd``` and ```virtlogd```. Do you want to cover them ?

Reply:
virtlogd is required by other daemons and will be started automatically. It seems that virtproxyd need be restarted manually at here?

Reply:
Short update,  confirm that virtlogd and virtlockd without virtproxyd-ro.socket and virtlockd-admin.socket. 
meanwhile, virtproxyd have not this problem:
```
# systemctl status virtproxyd
virtproxyd-admin.socket  virtproxyd.service       virtproxyd-tcp.socket    
virtproxyd-ro.socket     virtproxyd.socket        virtproxyd-tls.socket
```

FYI
```
# systemctl status virtlogd
virtlogd-admin.socket  virtlogd.service       virtlogd.socket
# systemctl status virtlockd
virtlockd-admin.socket  virtlockd.service       virtlockd.socket
```